src/style.css

```diff
@@ -1 +1,32 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 22px;
```

Comment:
It's not a good practice to use add such styles for body element. It's a common container for the whole web page, so the only styles which make sense for body are: the ones for resetting browser's default styles for body (`margin: 0` in this case), and some general styles applied to all elements on the page (like font size, text color etc.).
In you case it's better to wrap your `stars` blocks in another container block, and define styles for this block